bindings/python/iota_sdk/client/_node_core_api.py

```diff
@@ -50,107 +52,362 @@ def get_health(self, url: str):
             'url': url
         })
 
-    def get_node_info(self, url: str, auth=None) -> NodeInfo:
-        """Get node info.
+    # TODO: this is not strictly following the 2.0 Core API Spec (or maybe the TIP isn't updated yet)
+    # https://github.com/iotaledger/iota-sdk/issues/1921
+    def get_info(self) -> NodeInfoWrapper:
+        """Returns general information about the node together with its URL.
+        GET /api/core/v3/info
+        """
+        return NodeInfoWrapper.from_dict(self._call_method('getInfo'))
+
+    # TODO: this is not strictly following the 2.0 Core API Spec (or maybe the TIP isn't updated yet)
+    # https://github.com/iotaledger/iota-sdk/issues/1921
+    def get_node_info(self, url: str, auth=None) -> InfoResponse:
+        """Returns general information about the node.
+        GET /api/core/v3/info
 
         Args:
             url: The node's url.
             auth: A JWT or username/password authentication object.
+
+        Returns:
+            The node info.
         """
-        return NodeInfo.from_dict(self._call_method('getNodeInfo', {
+        return InfoResponse.from_dict(self._call_method('getNodeInfo', {
             'url': url,
             'auth': auth
         }))
 
-    def get_info(self) -> NodeInfoWrapper:
-        """Return node information together with the url of the used node.
+    # TODO: this should made be available
+    # https://github.com/iotaledger/iota-sdk/issues/1921
+    def get_routes(self) -> RoutesResponse:
+        """Returns the available API route groups of the node.
+        GET /api/routes
+        """
+
+    # def get_tips(self) -> List[BlockId]:
```

Comment:
What's up with all this ?

Reply:
good catch ... happened during the merge :disappointed: